alpine-rails-buildkite/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM dkubb/alpine-rails-nginx
+
+# Improve root state, intend to be pushed there
+WORKDIR /
+
+COPY etc /etc
+
+RUN rm -r /etc/sv/rails /etc/service/rails \
+  && /root/setup-directories.sh root r /etc/service/postgres /etc/service/buildkite \
+  && /root/setup-directories.sh postgres rw /var/db/postgresql \
+  && ln -s -- /etc/sv/postgres /etc/service/postgres/run \
+  && ln -s -- /etc/sv/buildkite /etc/service/buildkite/run \
+  && echo '@testing-extra https://s3.amazonaws.com/alpine-packages/testing-extra' >> /etc/apk/repositories \
+  && apk add --update-cache imagemagick buildkite-agent@testing-extra phantomjs@testing-extra
+
+USER buildkite
+
+# Setup bundler for buildkite user
+RUN bundle config --global build.nokogiri '--use-system-libraries' \
+  && bundle config --global jobs '8'
+
+USER postgres
+
+# Setup user and database
+RUN pg_ctl initdb --pgdata /var/db/postgresql/data \
```

Comment:
I wonder if you could do `export PGDATA='/var/db/postgresql/data'` as the first statement in `RUN`, and then skip specifying `--pgdata` in all these commands?